src/switcher-ui.tsx

```
@@ -1372,6 +1558,15 @@ function SwitcherApp() {
                   clearSessionSearchOnShowRef.current = true;
                   window.electronAPI.openClaudeSession(s.sessionId, s.project, s.isActive, s.activePid, customTitles[s.sessionId]);
                 }
+              } else if ((e.metaKey || e.ctrlKey) && (e.key === 'd' || e.key === 'D')) {
```

ESLint

[error] Replace `··············}·else·if·((e.metaKey·||·e.ctrlKey)·&&·(e.key·===·'d'·||·e.key·===·'D')` with `··················}·else·if·(⏎····················(e.metaKey·||·e.ctrlKey)·&&⏎····················(e.key·===·'d'·||·e.key·===·'D')⏎··················`

(prettier/prettier)